.gitignore

```diff
@@ -50,7 +50,13 @@
 .env
 
 ## docker compose file
-docker-compose.yml
+# docker-compose.yml
```

Comment:
See my comment below about keeping this .gitignore'd. I prefer having it be excluded from the repo so that local config can be changed without committing.